src/link/index.tsx

```diff
@@ -14,6 +17,21 @@ const Link = React.forwardRef(
     const baseComponentProps = useBaseComponent('Link', {
       props: { color, external, fontSize, rel: props.rel, target: props.target, variant: props.variant },
     });
+
+    const analyticsMetadata: GeneratedAnalyticsMetadataLinkFragment = {
+      action: 'click',
+      detail: {
+        label: '',
+        href: props.href || '',
```

Comment:
Why do you cast to an empty string here? Missing href and empty attribute are both valid and distinct use-cases. I think they should be reported respectively

Reply:
I made it optional and set it only if present